job_definitions/notify_buyers_to_award_closed_briefs.yml

```diff
@@ -0,0 +1,81 @@
+{% set environments = ['production'] %}
+---
+{% for environment in environments %}
+{% with vars = [
+    {"offset": "28", "weeks": "4", "notify_template_id": "204d2bf5-5dd3-44c8-a68f-9111e040e831"},
+    {"offset": "56", "weeks": "8", "notify_template_id": "bc1f93a8-9529-48b6-9a82-af61bc6da332"}
```

Comment:
Do we need `offset` here? It feels cleaner to calculate it in the default as `(weeks * 7)`.

Reply:
Ooops, yes - David suggested that and I completely forgot 😸